package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "seek-style-guide",
-  "version": "11.2.0",
+  "version": "11.3.0",
```

Comment:
As with the other modules, let's keep the version bump as a separate PR once this functionality is merged